src/components/organisms/History.tsx

```diff
@@ -62,32 +62,34 @@ export default function History() {
 
   const device_id = "a7382f5c-3326-4cf8-b717-549affe1c2eb";
 
+  const updateDataState = (data: []) => {
+    const tempMax: number[] = [];
+    const tempMin: number[] = [];
+    const humidMax: number[] = [];
+    const humidMin: number[] = [];
+    const label: string[] = [];
+
+    data.forEach((day: any) => {
```

Comment:
Don't use `any`.
Make the type clear.
Reference the fastapi docs to know what kind of structure will be returned.

Reply:
Done